Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -0,0 +1,202 @@
+>
+>If you do not know the URL or don't want to use it at this time, enter a placeholder URL like `test://`. Assurance Session URL connection is still possible without app side URL configuration.
+>  
+> In Xcode the app URL can be configured using these steps:
+> 1. Select the project in the navigator.
+> 2. Select the app target in the `Targets` section, in the project configuration window.
+> 3. Select the `Info` tab.
+> 4. Set the desired deep linking URL.
+> ![Xcode deeplink app url config](../assets/edge-bridge-tutorial/xcode-deeplink-app-url-config.jpg)
+> Please note that there is still code on the application side that is required for the app to respond to deep links; see the [guide on adding Assurance to your app](https://aep-sdks.gitbook.io/docs/foundation-extensions/adobe-experience-platform-assurance#add-the-aep-assurance-extension-to-your-app). For general implementation recommendations and best practices, see Apple's guide on [Defining a custom URL scheme for your app](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app)
+
+When presented with this window, the new Assurance session is created, and it is now possible to connect the app to your Assurance session.  
+<img src="../assets/edge-bridge-tutorial/assurance-create-session-qr.png" alt="Creating a new session in Assurance step 3 - QR code" width="400"/>
+<img src="../assets/edge-bridge-tutorial/assurance-create-session-link.png" alt="Creating a new session in Assurance step 3 - Session link" width="400"/>
+
+### 2. Connect to the app   
+There are two primary ways to connect an app instance to an Assurance session:
+1. QR Code: available with `Scan QR Code` option selected. Only works with physical devices, as it requires a physical device's camera to scan the code. Note that this method requires setup on the application code side to allow for deep linking (see [Set up the Assurance session](#1-set-up-the-assurance-session)).
+
+2. Session Link: available with `Copy Link` option selected. Works with both physical and simulated devices.
+
+To access these connection methods, click `Session Details`:  
+<img src="../assets/edge-bridge-tutorial/assurance-session-details-qr.png" alt="Assurance Session Details - QR code" width="400"/>
+<img src="../assets/edge-bridge-tutorial/assurance-session-details-link.png" alt="Assurance Session Details - Session link" width="400"/>
+
+Note that it is possible to edit both the `Session Name` and `Base URL`; changes to the `Base URL` value will automatically be reflected in both QR code and session link.
+
+
+### 3. Event transactions view - check for Edge Bridge events  
+After creating a new session, by default Assurance should open to the Event transactions view in the left side navigation panel: `Home` -> `Events`. This view shows you all of the events captured by Assurance after a connection with the app is established.
```

Comment:
Instead of default Events view, we can provide steps and screenshots from the Event transactions view (left nav Adobe Experience Platform Edge -> Event Transactions)